em_workflows/lrg_2d_rgb/flow.py

```diff
@@ -71,39 +71,42 @@ def bioformats_gen_zarr(file_path: FilePath):
         output_zarr,
     ]
     FilePath.run(cmd, log_fp)
-    cmd = ["zarr_rechunk", "--chunk-size", "512", output_zarr]
-    log_fp = f"{file_path.working_dir}/{file_path.base}_rechunk.log"
-    FilePath.run(cmd, log_fp)
+    zarr_images = HedwigZarrImages(zarr_path=Path(output_zarr), read_only=False)
+    zarr_image: HedwigZarrImage = next(zarr_images.series())[1]
```

Comment:
this line is a bit cryptic - can we write a reminder that the `.series()` method is returning a tuple, the second elt of which is the object, thus requiring us to walk one element in (and ignore the first one)? I remember this now, but I'm likely to forget this later...

Reply:
This could alternatively be something like: `zarr_image = zarr_images[zarr_images.get_series_keys()[0]]`

Reply:
Yes Mike. Although it was clear enough, I thought maybe typing can help code navigation.

Reply:
yeah, I like the type hinting. It's cute.

Reply:
I agree! It's better if used in a controlled manner.